File: Semester 4/Programming and Projection Enironments/Proiect_MPP/src/main/java/root/proiect_mpp/repositories/people/employees/IEmployeeRepository.java
package root.proiect_mpp.repositories.people.employees;

import root.proiect_mpp.domain.people.Employee;
import root.proiect_mpp.repositories.people.IPersonRepository;

import java.util.Collection;

public interface IEmployeeRepository extends IPersonRepository<Employee> {
    Employee findByEmail(String email);
    Collection<Employee> getByPosition(String position);
}

File: Semester 4/Programming and Projection Enironments/mpp-proiect-csharp/Proiect_MPP/Proiect_MPP/repository/people/employees/IEmployeeRepository.cs
using Proiect_MPP.domain.people;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proiect_MPP.repository.people.employees
{
    public interface IEmployeeRepository : IPersonRepository<Employee>
    {
        Employee? findByEmail(string email);
        List<Employee> getByPosition(string position);
    }
}
